modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/DropdownMenu.js

```diff
@@ -40,7 +40,7 @@ export default function DropdownMenu({
 				className={classNames({
 					'dropdown-action': actionsDropdown,
 				})}
-				items={normalizeDropdownItems(items)}
+				items={normalizeDropdownItems(items) || []}
```

Comment:
Why are we not using `useMemo` here following the same pattern?

Reply:
Dropdown doesn't have any external state so it won't rerender AFAIK

Reply:
<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia3.giphy.com%2Fmedia%2F3oEjI5VtIhHvK37WYo%2Fgiphy.gif&token=nice" data-gitmeme-token="nice"><img src="https://media3.giphy.com/media/3oEjI5VtIhHvK37WYo/giphy.gif" title="Created by gitme.me with /nice"/></a>